app/models/supermarket.rb

```diff
@@ -1,3 +1,4 @@
 class Supermarket < ApplicationRecord
   has_many :items
+  has_many :customers
```

Comment:
For this additional association to work, you'd need `has_many :customers, through: :items` since this is true for supermarket only because of its relationship to items